backend/crud.py

```
@@ -314,3 +315,184 @@ def delete_spot(db: Session, spot_id: UUID, user_id: UUID) -> None:
+    db.commit()
+    db.refresh(quest)
+    return get_quest_by_id(db, quest.id)
+
+
+def accept_quest(
+    db: Session,
+    quest_id: UUID,
+    walker_id: UUID,
+    distance_at_accept_m: Optional[int] = None,
+) -> models.Quest:
+    """クエストを受注（参加者レコードを作成/更新）"""
```

Ruff (0.14.8)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)